final_assignment/alex-mitan/conf.py

```diff
@@ -0,0 +1,161 @@
+#!/usr/bin/env python
+# *-* coding: UTF-8 *-*
+
+from __future__ import print_function
+
+import ast
+import os
+import sys
+import time
+import urllib
+
+# the raw content of the config file
+
+build_log = open("build.log", "w")
+
+
+def func_config(cfg_path):
+    # Phase -1 - Set variables
+    build_log.write("Working platform: " + sys.platform + '\n')
```

Comment:
Revenind ce ziceam mai sus cu modularitatea, logul ar fi fost un exemplu ideal de functie. Poti de exemplu sa folosesti o functie: write_log() care se ocupa ea de scrisul in log si doar o apelezi cand ai ceva de scris. Nu mai ai nevoie sa te plimbi cu fisierul prin tot codul. Daca vrei sa fi si mai smecher, poti sa faci functia asta sa scrie diferite niveluri de log, de exemplu: info, error etc.